components/console_simple_init/CHANGELOG.md

```diff
@@ -0,0 +1,9 @@
+# Changelog
+
+## [0.0.1]()
```

Comment:
Was this file generated by the script? I don't see the `bump` commit

Reply:
I manually created it.

Reply:
also, this should point to the commit id: https://github.com/espressif/esp-protocols/commits/123456)